composer.json

```diff
@@ -4,7 +4,7 @@
     "type": "magento2-module",
     "license": "MIT",
     "require": {
-        "omise/omise-php": "dev-master"
```

Comment:
First, I got an error like, https://github.com/composer/composer/issues/5211
Then, I followed this suggestion http://stackoverflow.com/questions/21052831/how-to-solve-two-packages-requirements-conflicts-when-running-composer-install/21467982#21467982